jerry-core/jcontext/jcontext.c

```diff
@@ -19,6 +19,7 @@
  * @{
  */
 
+#ifndef JERRY_ENABLE_EXTERNAL_CONTEXT
```

Comment:
I would like special cases first, and default case last. When we have multiple context types, that will look better.